lib/asciidoctor/path_resolver.rb

```diff
@@ -135,16 +135,17 @@ def initialize file_separator = nil, working_dir = nil
   # This operation considers posix paths, windows paths, and file URLs.
   #
   # Unix absolute paths and UNC paths start with slash. Windows roots can start
-  # with a drive letter. Absolute paths in the browser start with file://.
+  # with a drive letter. Absolute paths may start with file://, http://, or
+  # https:// when the IO module is xmlhttprequest (Opal runtime only).
   #
   # path - the String path to check
   #
   # returns a Boolean indicating whether the path is an absolute root path
   if RUBY_ENGINE == 'opal'
     def root? path
       (path.start_with? SLASH) ||
-          (::JAVASCRIPT_IO_MODULE == 'xmlhttprequest' && (path.start_with? 'file://')) ||
-          (@file_separator == BACKSLASH && (WindowsRootRx.match? path))
+          (@file_separator == BACKSLASH && (WindowsRootRx.match? path)) ||
+          (::JAVASCRIPT_IO_MODULE == 'xmlhttprequest' && (path.start_with? 'file://', 'http://', 'https://'))
```

Comment:
I didn't know that `start_with?` accepted a list of arguments! It's good to know :smile:

Reply:
I only discovered that recently and was like :champagne: